client/src/api/user-data.ts

```diff
@@ -0,0 +1,73 @@
+import { Round, UserGameResponse, AnswerResponse, RoundResponse } from "server";
+
+export const UserDataApi = {
+  startGame: (uuid: string = undefined): Promise<UserGameResponse | Error> => {
```

Comment:
```suggestion
  startGame: async (uuid: string = undefined): Promise<UserGameResponse | Error> => {
```

☝️ you can put async here, and then you don't have to wrap anything in Promise below.